ENIGMAsystem/SHELL/Universal_System/Extensions/FileDropper/FileDropper.cpp

```diff
@@ -1,4 +1,4 @@
-/** Copyright (C) 2019 Samuel Venable
+/** Copyright (C) 2020 Samuel Venable
```

Comment:
You don't just replace the copyright year, unless 2019 was originally a mistake and you did in fact write this entire `FileDropper.cpp` source in the year 2020. When you've made significant changes, you add a comma and amend the new year. Multiple consecutive years can also be combined with a dash e.g, `2019-2021` as opposed to listing each year. Do NOT think that this means you can apply copyright to years in which the source code was not edited. `2019-2021` would not be correct if you didn't change the source in 2020, in that case it would and should be `2019, 2021`.